WordPress/Classes/ViewRelated/Stats/Charts/StatsBarChartView.swift

```diff
@@ -110,10 +109,17 @@ class StatsBarChartView: BarChartView {
 
         if let dataSets = barChartData.dataSets as? [BarChartDataSet] {
             for dataSet in dataSets {
+                dataSet.colors = [ styling.barColor ]
+
                 dataSet.drawValuesEnabled = false
-                dataSet.highlightEnabled = false
 
-                dataSet.colors = [ styling.barColor ]
+                if let barHilightColor = styling.highlightColor {
```

Comment:
Nitpick `barHilightColor` -> `barHighlightColor`?

Reply:
I will take care of this at once!